crates/bytecode/src/legacy/analysis.rs

```diff
@@ -4,6 +4,17 @@ use bitvec::{bitvec, order::Lsb0, vec::BitVec};
 use primitives::Bytes;
 use std::vec::Vec;
 
+/// Check if an opcode is terminating (will end execution)
+fn is_terminating_opcode(opcode: u8) -> bool {
+    // First check if it's a known opcode
+    if let Some(opcode_info) = opcode::OpCode::info_by_op(opcode) {
```

Comment:
We should only consider known opcodes. Comment from https://github.com/bluealloy/revm/issues/2796#issuecomment-3117233884